### ISSUE TITLE
wget broken in rancheros 1.1.3

### ISSUE THREAD

**Issue by raztiff** [rancher/os#2248]
**RancherOS Version: (ros os version)**
v1.1.3

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
AWS

**Issue**:
- Using `wget` on the default console busy box in rancher os version 1.1.3. 
- Trying to use `wget` to make standard web requests that responds with an unexpected error: `wget: error getting response: Connection reset by peer` 
- Problem only present for 1.1.3 version of busy box, tested with other consoles `ubuntu`, `centos` and using `docker` with `alpine` based image on a AWS instance running the current version discussed. 

This is crucial for our cattle labelling scheme, which is how we manage the cleanup of disconnected AWS instances, so is proving rather problematic and stopping us upgrading and using the latest C5 instance types.

This can even be proven using telnet from the same failing instance running busy box, which works successfully:
`(printf 'GET /latest/meta-data/instance-id HTTP/1.0\r\n'; sleep 1) | telnet 169.254.169.254 80`

**Example wget command (AWS static IP for Metadata service) FAILING:**
`wget -qO- http://169.254.169.254/latest/meta-data/placement/availability-zone`

**Known working version**
Previous known working rancher version: v1.1.0 

**Related issues**

wget is broken against AWS metadata: https://github.com/rancher/os/issues/2216
alpine:3.6 is unable to wget http://google.com: https://github.com/gliderlabs/docker-alpine/issues/344

**Comment by niusmallnan** [comment]
It has been fixed. You can try v1.2.0-rc2 or wait for the official v1.2.0.
```
$ ros -v
version v1.2.0-rc2 from os image rancher/os:v1.2.0-rc2

$ wget -O - -q http://169.254.169.254/latest/meta-data/instance-type
c5.large
```

**Comment by raztiff** [comment]
Thanks for the update!

Do we have a timeframe on the official 1.2.0 release, thanks again @niusmallnan

**Comment by niusmallnan** [comment]
We have shipped v1.2.0
